packages/builtin-components/src/components/spine.js

```diff
@@ -68,22 +67,10 @@ export default class Spine extends PureComponent {
             position: 'inside',
           },
         },
-        data: series.slice(1),
+        data: _.map(props.source, key),
       })),
-    };
-
-    const onEvents = {
-      click: args =>
-        this.props.onSlicerChange(_.defaults({}, {
-          dataObj: {
-            [_.first(this.props.source)[0]]: args.name,
-            [args.seriesName]: args.value,
-          },
-        }, args)),
-    };
+    },
+  })),
+);
 
-    return (
```

Comment:
add breakpoint here to log props

Make sure the pros are the same after your chage.